core/templates/dev/head/pages/exploration_editor/loading_modal_directive.html

```diff
@@ -0,0 +1,27 @@
+<script type="text/ng-template" id="modals/loading">
+  <div class="oppia-loading-body oppia-save-publish-loading">
```

Comment:
I'm looking at the definition of .oppia-save-publish-loading in the CSS file, and I don't see any reference to animated dots. Does removing it actually affect anything?